backend/entityservice/views/objectstore.py

```diff
@@ -0,0 +1,79 @@
+import json
+
+import opentracing
+from flask import request
+from minio.credentials import AssumeRoleProvider, Credentials
+
+from entityservice.settings import Config as config
+import entityservice.database as db
+from entityservice.object_store import connect_to_upload_object_store
+from entityservice.views import bind_log_and_span, precheck_upload_token
+from entityservice.views.serialization import ObjectStoreCredentials
+
+
+def _get_upload_policy(bucket_name="uploads", path="*"):
+
+    restricted_upload_policy = {
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Action": [
+            "s3:PutObject"
+          ],
+          "Effect": "Allow",
+          "Resource": [
+            "arn:aws:s3:::{}/{}".format(bucket_name, path),
+            "arn:aws:s3:::{}/{}/*".format(bucket_name, path),
+          ],
+          "Sid": "Upload-access-to-specific-bucket-only"
+        }
+      ]
+    }
+
+    return json.dumps(restricted_upload_policy)
+
+
+def authorize_external_upload(project_id):
+    assert config.UPLOAD_OBJECT_STORE_ENABLED, "Feature disabled"
```

Comment:
will that create an appropriate response for the request?

Reply:
I checked that the endpoint returns a `500` and the schema is correctly a `Problem`. However it doesn't actually send the "feature disabled" message to the user. I'll look into it.

Reply:
I forgot we had a `send_fail_request` for this exact purpose - fixed now.